script/run-unit-test.js

```diff
@@ -45,7 +45,7 @@ const mochaOpts =
 runCommand(
   `LOG_LEVEL=${options[
     'log-level'
-  ].toLowerCase()} ${testRunner} --opts ${mochaOpts} --recursive ${options.path
+  ].toLowerCase()} ${testRunner} --max-old-space-size=4096 --opts ${mochaOpts} --recursive ${options.path
```

Comment:
This makes `yarn test:unit` run the full suite without running out of RAM

Reply:
@jbalboni might want to be aware of this

Reply:
For the record, it looks like this won't affect anything in Jenkins.

Reply:
Interesting. For now that's fine but it seems like only matter of time before the entire suite no longer runs in Jenkins either.